notebooks/measures/Search_Download_and_Plot_GeoTIFFs.ipynb

```diff
@@ -0,0 +1,503 @@
+    "\n",
+    "We will look at two data sets focused on Greenland: a Digital Elevation Model (DEM) and velocity at glacier sites ([NSIDC-0715](https://nsidc.org/data/nsidc-0715) and [NSIDC-0481](https://nsidc.org/data/nsidc-0481)). We will use the `search_data` method from the `earthaccess` library and the following variables to search for granules within these data sets and it will output the number of granules that meet the search criteria for each data set:\n",
+    "\n",
+    "* `short_name` - this is the data set ID e.g., NSIDC-0715, NSIDC-0481. It can be found in the data set title on the data set landing page.\n",
+    "* `version` - data set version number, also included in the data set title.\n",
+    "* `cloud_hosted` - NSIDC is in the process of migrating data sets to the cloud. The data sets we are interested in are currently still archived on-prem so will set this to False.\n",
+    "* `bounding_box` - sets a spatial filter by specifying latitude and longitude in the following order: W, S, E, and N.\n",
+    "* `temporal` - sets a temporal filter by specifying a start and end date in the format YYYY-MM-DD.\n",
+    "* `count` - this sets the maximum number of granules that will be returned in the search\n",
+    "\n",
+    "**NOTE** The data sets and the spatial and temporal filters used here are just examples. If there are other data sets in GeoTIFF format or different spatial and temporal filters we wish to try, then we can just modify the `short_name`, `bounding_box` and `temporal` variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62b87b2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Search for DEM files\n",
+    "print('For NSIDC-0715')\n",
+    "results_dem = earthaccess.search_data(\n",
+    "    short_name='NSIDC-0715',\n",
+    "    version='2',\n",
+    "    cloud_hosted=False,\n",
+    "    bounding_box=(-33.45,68.29,-31.41,69.26),\n",
+    "    temporal=('2015-12-01','2015-12-31'),\n",
+    "    count=100\n",
+    ")\n",
+    "\n",
+    "#Search for velocity data \n",
+    "print('For NSIDC-0481')\n",
+    "results_vel = earthaccess.search_data(\n",
+    "    short_name='NSIDC-0481',\n",
+    "    version='4',\n",
+    "    cloud_hosted=False,\n",
+    "    bounding_box=(-33.45,68.29,-31.41,69.26),\n",
+    "    temporal=('2017-01-01','2017-12-31'),\n",
+    "    count=100\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f58d1b42",
+   "metadata": {},
+   "source": [
+    "### **Download the data**\n",
+    "Now we have found granules that meet our search criteria we can download them using the `download` method from `earthaccess`. Note that for these particular data sets within each granule there are multiple files. So even though 1 granule was found for the DEM data set, 6 files will be downloaded. The progress and size of each order will be printed out.\n",
```

Comment:
This is a great explanation of files vs granules. It may be helpful to quickly explain what those 6 files are? Just to understand the files per granule structure a bit more?

Reply:
Good idea, have made that change.